pkg/api/pod/warnings.go

```diff
@@ -168,6 +168,56 @@ func warningsForPodSpecAndMeta(fieldPath *field.Path, podSpec *api.PodSpec, meta
 		}
 	}
 
+	items := sets.NewString()
+	for _, v := range podSpec.Volumes {
+		if v.Projected != nil {
+			for _, k := range v.Projected.Sources {
+				if k.ServiceAccountToken != nil {
+					items.Insert(k.ServiceAccountToken.Path)
+				}
+			}
+		}
+
+	}
+
+	if items.Len() > 0 {
+		for i, item := range podSpec.Volumes {
+			if item.Projected != nil {
+				for _, k := range item.Projected.Sources {
+					if k.ConfigMap != nil {
+						for j, item := range k.ConfigMap.Items {
+							if items.Has(item.Path) {
+								warnings = append(warnings, fmt.Sprintf("%s: has duplicated path with ServiceAccountToken %q", fieldPath.Child("spec", "volumes").Index(i).Child("projected", "sources", "configMap").Index(j), item.Path))
```

Comment:
This doesn't help if the Items field is not specified, right?  It is strictly opional and defaults to "all keys in the map"

Reply:
Fixed.